internal/provisioner/plain/README.md

```diff
@@ -125,52 +131,44 @@ Create the bundle:
 ```console
 $ kubectl apply -f -<<EOF
 apiVersion: core.rukpak.io/v1alpha1
-kind: Bundle
+kind: BundleInstance
 metadata:
-  name: combo-v0.0.1
+  name: combo
 spec:
-  source:
-    type: image
-    image:
-      ref: quay.io/tflannag/bundle:combo-operator-v0.0.1
   provisionerClassName: core.rukpak.io/plain
+  selector:
+    matchLabels:
+      subscription: combo
+  template:
+    metadata:
+      labels:
+        subscription: combo
+    spec:
+      provisionerClassName: core.rukpak.io/plain
+      source:
+        git:
+          ref:
+            tag: v0.0.1
+          repository: https://github.com/operator-framework/combo
+        type: git
 EOF
-bundle.core.rukpak.io/combo-v0.0.1 created
-```
-
-Check the Bundle status via `kubectl get bundle combo-0.0.1`. Eventually the Bundle should show up as Unpacked.
-
-```console
-$ kubectl get bundle combo-v0.0.1
-NAME           IMAGE                                           PHASE      AGE
-combo-v0.0.1   quay.io/tflannag/bundle:combo-operator-v0.0.1   Unpacked   10s
+bundleinstance.core.rukpak.io/combo created
 ```
 
-Create the combo `BundleInstance` referencing the combo `Bundle` available in the cluster.
-
-Note: it's perfectly valid to create a `Bundle` and `BundleInstance` referencing that `Bundle` at the same time. There
-is no specific ordering required, which is perfect for GitOps flows, and eventually both the `Bundle`
-and `BundleInstance` would succeed.
+Check the Bundle status via `kubectl get bundle -l subscription=combo`. Eventually the Bundle should show up as Unpacked.
 
 ```console
-$ kubectl apply -f -<<EOF
-apiVersion: core.rukpak.io/v1alpha1
-kind: BundleInstance
-metadata:
-  name: combo
-spec:
-  provisionerClassName: core.rukpak.io/plain
-  bundleName: combo-v0.0.1
-EOF
-bundleinstance.core.rukpak.io/combo created
+$ kubectl get bundle -l subscription=combo
+NAME          TYPE   PHASE      AGE
+combo-9njsj   git    Unpacked   10s
 ```
 
 Check the BundleInstance status to ensure that the installation was successful.
 
 ```console
 $ kubectl get bundleinstance combo
-NAME    DESIRED BUNDLE   INSTALLED BUNDLE   INSTALL STATE           AGE
-combo   combo-v0.0.1     combo-v0.0.1       InstallationSucceeded   10s
+NAME    INSTALLED BUNDLE   INSTALL STATE           AGE
+combo   combo-9njsj        InstallationSucceeded   10s
```

Comment:
Should we add a printerColumn that indicates what ref this `BundleInstance` is referencing? It isn't immediately obvious what we're referencing with this `BundleInstance` without going to the created Bundle itself. Not important but I could see a situation like this:

```console
NAME    INSTALLED BUNDLE   INSTALL STATE           REF      AGE
combo   combo-9njsj        InstallationSucceeded   v0.0.1   10s
```

In essence it is very similar to https://github.com/operator-framework/rukpak/pull/276

Reply:
I think this is worth chasing but I think we'll end up overhauling the BI's status (and therefore implicitly the printer columns) relatively soon, and I think we'll end up mirroring the deployment printer columns where it uses booleans vs. strings to expose state.